### PR TITLE
-> v0.6.2: multiple expected scopes asks for one match instead of all…

### PR DESCRIPTION
**Pull request by weiyunlu**
*Related Issue(s) or Task(s)*

https://app.zenhub.com/workspaces/tech-kanban-5f3b11686371ae001e21fc4b/issues/zetatango/zetatango/12024

*Why?*

Change behaviour on `valid_scope?` to take disjunction instead of conjunction of `expected_scopes` so that we can support multiple scopes on an API endpoint.

Was to be applied to Leads API, but since it sounds like this was something we want to do in general and the work was already done, might as well salvage something out of my latest Adventure in the Realm™.

Incidentally, also there is a security vulnerability with actionview 6.0.3.3 so update it.

*How?*

Change logic of `valid_scope`.

Run `bundle update actionview`.

*Risks*

Low, AFAIK we don't have any endpoints that demand multiple scopes.

*Requested Reviewers*

Brent, Dragos

### CODE REVIEW

**Review by dragoszt — APPROVED** [review]
lgtm

File: lib/token_validator/token_service.rb [hunk 1]
```diff
@@ -52,12 +52,12 @@ def valid_structure?
   def valid_scope?
     raise InvalidScope, 'Missing scopes' unless decoded_jwt.key?('scopes')
 
-    valid = true
+    valid = false
```
Comment (bcarr092) [inline lib/token_validator/token_service.rb:55]: Yikes, we should have always defaulted to `false`!

**Review by bcarr092 — APPROVED** [review]

File: lib/token_validator/token_service.rb (hunk 1 above)
Comment (weiyunlu) [reply lib/token_validator/token_service.rb:55]: Innocent until proven guilty?  :P

Comment (dragoszt) [reply lib/token_validator/token_service.rb:55]: I almost commented on this and then realized is the old code

**Pull request merged**